### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NotebookQueryTable_ColorContrast: Color contrast ratio of "sys.tables"text for Solarized dark theme is below the expected ratio i.e 4.5:1

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5978]
[“Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

**Additional Details:**
Tool Name: ColorContrastAnalyser
MAS Reference: MAS1.4.3

**Steps to Reproduce:**

1. Install the app Azure Data Studio.
2. Connect to the Database->Go to file->New Notebook.
3. Click on “Code” button and write query in the box for example SELECT * FROM sys.tables;
4. Go to settings and select color theme and apply Soalrized Dark theme.
5. Open color contrast analyser and check the color contrast ratio.
6. It shows color contrast ratio less than expected color contrast ratio.

**Actual:**
Color contrast ratio for "sys.tables" text is 2.986:1.
And similar issue also observed with "select * from " text is 3.134:1 and for "1" text is 3.614:1 .

**Expected:**
The expected color contrast ratio  for text should be 4.5:1 ratio.

**User Impact:**
Low vision User will find it difficult to understand and will be confused if they will not able to see the buttons on this page.

**Recommendations:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:
MAS1.4.3-** https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba73546c7-765f-489c-b18f-afb659fe99e6%7D

**Attachment For Reference:**
![5978_ Accessibility A11y_AzureDataStudio_NotebookQueryTable_ColorContrast](https://user-images.githubusercontent.com/39204909/59269851-9889c380-8c6d-11e9-85e9-26df2b813e36.png)
![5978_ Accessibility A11y_AzureDataStudio_NotebookQueryTable_ColorContrast1](https://user-images.githubusercontent.com/39204909/59269852-99225a00-8c6d-11e9-9fa8-cfb058218ad4.png)
![5978_ Accessibility A11y_AzureDataStudio_NotebookQueryTable_ColorContrast2](https://user-images.githubusercontent.com/39204909/59269853-99225a00-8c6d-11e9-831a-352738ae9874.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chlafreniere** [comment]
Closing after conversation with accessibility testing team (no contrast requirements on non-default themes).
